code/core/src/instrumenter/instrumenter.test.ts

```diff
@@ -712,6 +730,40 @@ describe('Instrumenter', () => {
     });
   });
 
+  describe('while detached from parent window', () => {
+    beforeEach(() => {
+      global.window.parent = {
+        ...global.window.parent,
+        get __STORYBOOK_ADDON_INTERACTIONS_INSTRUMENTER_STATE__() {
+          throw new Error('Blocked');
+        },
+        set __STORYBOOK_ADDON_INTERACTIONS_INSTRUMENTER_STATE__(_: unknown) {},
```

Comment:
Long term, would be great to stop having to do this somehow.

Accessing `global.window.parent` should never be done.
Surely this can be done using the channel?

Till then this looks like an improvement to me.